openedx/core/djangoapps/contentserver/middleware.py

```diff
@@ -211,7 +211,7 @@ def set_caching_headers(self, content, response):
                 newrelic.agent.add_custom_parameter('contentserver.cacheable', True)
 
             response['Expires'] = StaticContentServer.get_expiration_value(datetime.datetime.utcnow(), cache_ttl)
-            response['Cache-Control'] = b"public, max-age={ttl}, s-maxage={ttl}".format(ttl=cache_ttl)
+            response['Cache-Control'] = "public, max-age={ttl}, s-maxage={ttl}".format(ttl=cache_ttl)
```

Comment:
You should prefix it with a `u` to make it clear that it's a unicode string. That should fix the quality error as well.